client/src/components/application/NonlinearStepper/GeneralInfo.js

```diff
@@ -87,6 +159,65 @@ const GereralInfos = ({ handleFormInput, formValues, errorMsg }) => {
         value={formValues.address}
         isError={errorMsg.includes('Address is required')}
       />
+      <SelectBox
+        className="nonlinearSelect"
+        label="university"
+        name="university"
+        items={
+          formValues.location === 'gaza' ? GazaUniversity : KhalilUniversity
+        }
+        setVal={handleFormInput}
+        value={formValues.university}
+        isError={errorMsg.includes('university')}
+      />
+      <InputText
+        className="nonlinearSelect"
+        value={formValues.specialization}
+        placeholder="If you went to university, what was your specialization?"
+        label="specialization"
+        name="specialization"
+        onChange={handleFormInput}
+        isError={errorMsg.includes('specialization')}
+        message="(If you didn't go to university, just write N/A) ?"
+      />
+      <SelectBox
+        className="nonlinearSelect"
+        label="Coding Experience"
+        name="codingExperience"
+        items={codeExperience}
+        setVal={handleFormInput}
+        value={formValues.codingExperience}
+        isError={errorMsg.includes('codingExperience')}
+      />
+      <SelectBox
+        className="nonlinearSelect"
+        label="Primary Motivation"
+        name="primaryMotivation"
+        items={primaryMotivationForm}
+        setVal={handleFormInput}
+        value={formValues.primaryMotivation}
+        isError={errorMsg.includes('primaryMotivation')}
+      />
+      <SelectBox
+        className="nonlinearSelect"
+        label="How did you hear about the Code Academy"
+        name="caReading"
+        items={advert}
+        setVal={handleFormInput}
+        value={formValues.caReading}
+        isError={errorMsg.includes('caReading')}
+      />
+      <InputText
+        className="nonlinearSelect"
+        value={formValues.cvLink}
+        placeholder="Link for latest version of your CV in English"
+        label="CV Link"
+        name="cvLink"
+        onChange={handleFormInput}
+        isError={
+          errorMsg.includes('cvLink') || errorMsg.includes('Enter correct url!')
+        }
+      />
```

Comment:
for the file, we have to add file as attachment, so input should be from file type 
<input type="file" size="60" /> for example.